### PR TITLE
Add tokenize_kwargs parameter definition in the FeatureExtractionPipeline

### LINKED ISSUE #21971: No clear documentation for enabling padding in FeatureExtractionPipeline
### System Info

- `transformers` version: 4.26.0
- Platform: Linux-4.19.0-23-cloud-amd64-x86_64-with-debian-10.13
- Python version: 3.7.12
- Huggingface_hub version: 0.12.0
- PyTorch version (GPU?): 1.13.1+cu116 (True)
- Tensorflow version (GPU?): not installed (NA)
- Flax version (CPU?/GPU?/TPU?): not installed (NA)
- Jax version: not installed
- JaxLib version: not installed
- Using GPU in script?: <fill in>
- Using distributed or parallel set-up in script?: <fill in>

### Who can help?

@Narsil

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

The documentation for the [FeatureExtractionPipeline](https://huggingface.co/docs/transformers/v4.26.1/en/main_classes/pipelines#transformers.FeatureExtractionPipeline)  does not currently provide instructions on how to use the truncation and padding arguments. 

These arguments can be passed in the `tokenize_kwargs` parameter, which is parsed by the [self._sanitize_parameters](https://github.com/huggingface/transformers/blob/v4.26.1/src/transformers/pipelines/feature_extraction.py#L58) method. While the `truncation` argument can also be passed as a separate keyword argument, the `padding` argument can only be recognized if it is included in `tokenize_kwargs`.

To improve clarity, it would be beneficial for the documentation to explicitly state that the existence of `tokenize_kwargs` parameter for passing tokenizer arguments and add that only the `truncation` argument can be used as a keyword argument, while other tokenizer parameters should be included in `tokenize_kwargs`.

### Expected behavior

Below is the code to show that how padding should be used in FeatureExtractionPipeline.

```python
example = "After stealing money from the bank vault, the bank robber was seen fishing on the Mississippi river bank."
tokenizer = AutoTokenizer.from_pretrained("bert-base-uncased", use_fast=False)

pipeline_without_padding_as_an_argument = pipeline(
    "feature-extraction",
    model="bert-base-uncased",
    tokenizer=tokenizer,
    return_tensors=True,
)
pipeline_with_padding_as_an_argument = pipeline(
    "feature-extraction",
    model="bert-base-uncased",
    tokenizer=tokenizer,
    padding="max_length",
    return_tensors=True,
)

pipeline_with_padding_in_kwarg = pipeline(
    "feature-extraction",
    model="bert-base-uncased",
    tokenizer=tokenizer,
    tokenize_kwargs={"padding": "max_length"},
    return_tensors=True,
)

print(
    pipeline_without_padding_as_an_argument(example).shape
)  # torch.Size([1, 22, 768])
print(
    pipeline_with_padding_as_an_argument(example).shape
)  # torch.Size([1, 22, 768]) padding = max_length not working
print(
    pipeline_with_padding_in_kwarg(example).shape
)  # torch.Size([1, 512, 768]) padding = max_length working
```

### PR DESCRIPTION
**Pull request by anruijian**
# What does this PR do?

Fixes #21971 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@Narsil

### CODE REVIEW

**Comment by Narsil** [comment]
Looks great to me.

To fix quality you can do `pip install -e .[quality] && make fixup` 

@sgugger for final review.
